dashboard/app/controllers/scripts_controller.rb

```diff
@@ -145,18 +145,22 @@ def instructions
 
   def vocab
     return render :forbidden unless can? :read, @script
+    render 'vocab', locals: {unit_summary: @script.summarize_for_rollup(@current_user)}
```

Comment:
please either set `@unit_summary` here, or skip entirely and just call `@script.summarize_for_rollup(@current_user)` from the haml file